### PR TITLE
Term lemmas

### PR DESCRIPTION
**Pull request by jessecoleman**
- show term lemmas in suggestions dropdown
- show recommended keywords in abstract dialog
- suggest topics to search for if no terms in input field
- overhaul input field styling to improve animations/transitions
- new steps in training pipeline to support new NLP features
- can now access grants dialog from FAB in bottom right
- sort terms alphabetically or by count

### CODE REVIEW

File: backend/app.py [hunk 1]
```diff
@@ -1,28 +1,34 @@
...
+from gensim.models.phrases import Phraser
 from gensim.models import KeyedVectors
 import logging
 import json
 from functools import lru_cache
 from aioelasticsearch import Elasticsearch
-from aioelasticsearch.helpers import Scan
 
 import queries as Q
+import analyze
```
Comment (h1-the-swan) [inline backend/app.py:22]: backend startup fails for me: `ModuleNotFoundError: No module named 'analyze'`

**Review by h1-the-swan — CHANGES_REQUESTED** [review]

File: backend/app.py (hunk 1 above)
Comment (jessecoleman) [reply backend/app.py:22]: should be fixed now

**Review by h1-the-swan — CHANGES_REQUESTED** [review]
Aside from the assets file that needs to be committed, it's looking good. Is there a way to turn off the new features to present the version from before?

**Review by h1-the-swan — APPROVED** [review]
Looking good! I removed the `nltk` dependency because it looks like we're not using it anymore.

**Pull request merged**